modules/account-lib/src/coin/hbar/coinTransferBuilder.ts

```diff
@@ -0,0 +1,133 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import * as Long from 'long';
+import * as proto from '@hashgraph/proto';
+import { InvalidParameterValueError } from '@bitgo/sdk-core';
+import { TransferBuilder, Recipient } from './transferBuilder';
+import { Transaction } from './transaction';
+import { isTokenTransfer, isValidAddress, isValidAmount, stringifyAccountId } from './utils';
+import { BigNumber } from 'bignumber.js';
+
+export class CoinTransferBuilder extends TransferBuilder {
+  // TODO: [BG-51282] Deprecate once wp work for multi recipients
+  private _toAddress: string;
+  private _amount: string;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+    this._recipients = [];
```

Comment:
There's no need to instantiate recipients here since the parent, `TransferBuilder`, takes care of it.